src/main/java/wromaciej/hvac_sim/thermo/Ids.java

```diff
@@ -0,0 +1,28 @@
+package wromaciej.hvac_sim.thermo;
+
+import wromaciej.hvac_sim.display.Item;
+
+import java.util.HashMap;
+import java.util.Set;
+
+public class Ids {
```

Comment:
definitely not. You are not able by any chance to test such code using that Id generator.

What I propose is doing interface `IdGenerator` with method `getUniqueId`. Then you will so far implement it with static incremented variable. When you are ready to use database, you will create next implementation.